torch/_decomp/__init__.py

```diff
@@ -220,9 +219,6 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten.frac,
             aten.frac_,
             aten._fused_moving_avg_obs_fq_helper,
-            aten.gelu,
-            aten.gelu_,
-            aten.gelu_backward,
```

Comment:
I'm not sure why these are removed? I don't see `gelu_backward` in the Core ATen IR: https://pytorch.org/docs/stable/ir.html

Reply:
Oops sorry, I was mistaken about this -- will put the x_ and x_backward ops back.